contracts/contracts/RaidenMicroTransferChannels.sol

```diff
@@ -31,8 +31,11 @@ contract RaidenMicroTransferChannels {
 
     // 28 (deposit) + 4 (block no settlement)
     struct Channel {
-        uint192 deposit; // mAX 2^192 == 2^6 * 2^18
-        uint32 open_block_number; // UNIQUE for participants to prevent replay of messages in later channels
+        // mAX 2^192 == 2^6 * 2^18
+        uint192 deposit;
+
+        // UNIQUE for participants to prevent replay of messages in later channels
```

Comment:
How is ensured that this is unique for participants?

Reply:
Replacing this with
```
// Used in creating a unique identifier for the channel between a sender and receiver.
// Supports creation of multiple channels between the 2 parties and prevents
 // replay of messages in later channels
```